src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/ContentCacheGenerationService.cs

```diff
@@ -0,0 +1,195 @@
+                new JsonSerializerSettings()
+                {
+                    ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
+                    NullValueHandling = NullValueHandling.Ignore
+                }));
+            return true;
+        }
+
+
+        private async Task<bool> UpdateMethodologies()
+        {
+            var methodologies = _methodologyService.Get();
+
+            foreach (var methodology in methodologies)
+            {
+                // TODO: model might be incorrect so will need to validate this
+                // TODO: Save the filename as slug rather than ID
+                var blob = _cloudBlobContainer.GetBlockBlobReference(
+                    $"methodology/methodologies/{methodology.Slug}.json");
+                await blob.UploadTextAsync(JsonConvert.SerializeObject(methodology, null,
+                    new JsonSerializerSettings() {ReferenceLoopHandling = ReferenceLoopHandling.Ignore}));
+            }
+
+            return true;
+        }
+
+        private async Task<bool> UpdatePublicationsAndReleases()
+        {
+            var publications = _publicationService.ListPublicationsWithPublishedReleases();
+
+            foreach (var publication in publications)
+            {
+                var publicationBlob =
+                    _cloudBlobContainer.GetBlockBlobReference($"publications/{publication.Slug}/publication.json");
+                await publicationBlob.UploadTextAsync(JsonConvert.SerializeObject(
+                    new PublicationViewModel() {Id = publication.Id, Title = publication.Title}, null,
+                    new JsonSerializerSettings() {ReferenceLoopHandling = ReferenceLoopHandling.Ignore}));
+
+                var latestRelease = _releaseService.GetLatestRelease(publication.Id);
+                var latestReleaseJson = JsonConvert.SerializeObject(latestRelease, null,
+                    new JsonSerializerSettings() {ReferenceLoopHandling = ReferenceLoopHandling.Ignore});
+
+                var latestReleaseBlob =
+                    _cloudBlobContainer.GetBlockBlobReference(
+                        $"publications/{latestRelease.Publication.Slug}/latest-release.json");
+                var latestReleaseYearBlob = _cloudBlobContainer.GetBlockBlobReference(
+                    $"publications/{latestRelease.Publication.Slug}/releases/{latestRelease.Slug}.json");
+
+                await latestReleaseBlob.UploadTextAsync(latestReleaseJson);
+                await latestReleaseYearBlob.UploadTextAsync(latestReleaseJson);
+
+                foreach (var r in publication.Releases)
+                {
+                    var release = _releaseService.GetRelease(r.Id);
+
+                    var releaseBlob =
+                        _cloudBlobContainer.GetBlockBlobReference(
+                            $"publications/{publication.Slug}/releases/{release.Slug}.json");
+
+                    // TODO: Fix model to ignore self referencing loop (quick workaround)
```

Comment:
Has this been fixed yet? It sounds like a serious problem rather than something that can be deferred.